### ISSUE TITLE
Can't display logout button

### ISSUE THREAD

**Issue by Josecanalla** [CakeDC/users#469]
Hello, I have a problem with permissions. 

When I do $this->User->logout(); the button to logout not appears. It is a problem of permissions.php, because if in my permissions I set:

```
[
            'role' => '*',
            'plugin' => '*',
            'controller' => '*',
            'action' => '*'
 ]
```

The button appears. But if I comment that and set:

```
[
            'role' => 'user',
            'plugin' => 'CakeDC/Users',
            'controller' => 'Users',
            'action' => ['register', 'edit', 'view', 'logout'],
            'allowed' => true
],
```

The button not appears. Why?

**Comment by steinkel** [comment]
There could be another rule affecting this permissions, or the user is not in "user" group. Please debug the rules matched in this case and let us know. Thanks!

**Comment by Josecanalla** [comment]
The problem is with the 'plugin' keyword. If I set
```
[
            'role' => 'user',
            'plugin' => 'CakeDC/Users',
            'controller' => '*',
            'action' => '*',
            'allowed' => true
]
```
I can't display the button. But if I set

```
[
            'role' => 'user',
            'plugin' => '*',
            'controller' => '*',
            'action' => '*',
            'allowed' => true
]
```
I can display it. Obviously, I try it with the same user logged in.

**Comment by steinkel** [comment]
Interesting use case, would be nice if you can provide a unit test to check this behaviour, we'll take a look anyway... thanks for the catch!

**Comment by Josecanalla** [comment]
I don't work with unit tests in CakePHP, sorry! :( . I will try another options and comment the results... thanks!

**Comment by steinkel** [comment]
Are you running your application inside a directory or do you have a virtualhost configured?

**Comment by Josecanalla** [comment]
Inside a directory

**Comment by steinkel** [comment]
This could be related to another issue (reverse routing inside a
subdirectory)

**Comment by steinkel** [comment]
closed per #543 please reopen if needed

**Comment by steinkel** [comment]
It was merged into master
